### DISCUSSION TITLE
Is it possible to use Poetry with meson-python?

### DISCUSSION THREAD

**Discussion by astro-stan** [mesonbuild/meson-python#637] [General]
I went over the docs but they don't show an example with poetry. Is this not supported? 

If it is supported could  you please provide me with a short tutorial, otherwise please point me to an alternative. Thanks!

**Comment by eli-schwartz** [comment]
IIRC poetry the workflow management tool, doesn't support any python build backends other than poetry-core. Something something lock-in. PDM apparently handles this fine.

**Reply by rgommers** [reply]
PDM should be one option indeed. Another is [spin](https://github.com/scientific-python/spin/), which projects like NumPy use for workflow automation. A key difference is that PDM (and Poetry) wants to handle your dependencies and setting up environments, while `spin` is agnostic to how you use your environment and focused on build/test/docs/lint/etc. automation.
